training/pytorch/structured/custom_containers/base/README.md

```diff
@@ -0,0 +1,75 @@
+# PyTorch Custom Containers Template
+
+## Overview
+
+The purpose of this directory is to provide a sample for how you can package a
+PyTorch training model to submit it to AI Platform. The sample makes it
+easier to organise your code, and to adapt it to your dataset. In more details,
+the template covers the following functionality:
+
+*   Metadata to define your dataset, along with the problem type (Classification).
+*   Standard implementation of input, parsing, and serving functions.
+*   Automatic feature columns creation based on the metadata (and normalization stats).
+*   Train, evaluate, and export the model.
+*   Parameterization of the experiment.
+
+Although this sample provides standard implementation to different
+functionality, you can customise these parts with your own implementation.
+
+## Prerequisites
+
+* Setup your project by following the instructions in the [setup](../../../../setup/) directory.
+* [Setup docker with Cloud Container Registry](https://cloud.google.com/container-registry/docs/pushing-and-pulling)
+* The datasets are downloaded by the Dockerfile.
+ * The Dockerfile defaults to downloading the small dataset, if you wish to modify this, you can set which files to download via the `--build-arg` flag:
+ ```
+ docker build -f Dockerfile -t $IMAGE_URI ./ --build-args train-files=GCS_PATH_TO_TRAIN_DATA eval-files=GCS_PATH_TO_EVAL_DATA
```

Comment:
Maybe add a sample? 
gcr.io/dpe-cloud-mle/pytorch
[HOSTNAME]/[PROJECT-ID]/[IMAGE]

Reply:
Also add sample for Datasets "gs://" path

Reply:
docker build -f Dockerfile -t gcr.io/dpe-cloud-mle/pytorch ./ --build-args train-files=gs://cloud-samples-data/ml-engine/chicago_taxi/training/small/taxi_trips_train.csv eval-files=gs://cloud-samples-data/ml-engine/chicago_taxi/training/small/taxi_trips_eval.csv
unknown flag: --build-args
See 'docker build --help'.

I needed to do:

docker build -f Dockerfile -t gcr.io/dpe-cloud-mle/pytorch ./ --build-arg train-files=gs://cloud-samples-data/ml-engine/chicago_taxi/training/small/taxi_trips_train.csv --build-arg eval-files=gs://cloud-samples-data/ml-engine/chicago_taxi/training/small/taxi_trips_eval.csv

Reply:
Docker Version 18.06.1-ce-mac73 (26764)

Reply:
Mis type. 
Updated